lib/suspenders/app_builder.rb

```diff
@@ -253,7 +253,7 @@ def configure_simple_form
     end
 
     def configure_action_mailer
-      action_mailer_host "development", %{"localhost:#{port}"}
+      action_mailer_host "development", %{ENV.fetch("APPLICATION_HOST")}
```

Comment:
I think we can use APPLICATION_HOST because it's already set to `localhost:3000` in `.sample.env`